src/ManagedServices/ManagedServices/help/Az.ManagedServices.md

```diff
@@ -0,0 +1,31 @@
+---
+Module Name: Az.ManagedServices
+Module Guid: {{ Update Module Guid }}
+Download Help Link: {{ Update Download Link }}
```

Comment:
Please follow the pattern in other module help files for these

Reply:
Module Name: Az.ManagedServices
Module Guid: d2a8f744-8dcb-4a13-ba80-eb181ff365ad
Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.managedservices
Help Version: 0.0.1
Locale: en-US